sdk/communication/communication-chat/test/chatClient.spec.ts

```diff
@@ -11,7 +11,7 @@ import { CommunicationIdentifier } from "@azure/communication-common";
 import { Context } from "mocha";
 
 describe("ChatClient", function() {
-  let threadId: string;
+  let threadId: string | undefined;
```

Comment:
Would it make sense to default this to an empty string or null? And `assert.isNotNull` later down in the test?

Reply:
Is this change being made so we can run in --strictNullChecks mode?

Reply:
Looks like we set the threadId in one of the tests and then use it in another. That makes one test dependent on another which is not ideal

Reply:
I think this is beyond the scope of this PR. But would be good to open an issue against it